src/Psalm/Internal/Provider/ReturnTypeProvider/ExplodeReturnTypeProvider.php

```diff
@@ -1,103 +0,0 @@
-<?php
-
-namespace Psalm\Internal\Provider\ReturnTypeProvider;
-
-use PhpParser;
-use Psalm\Internal\Analyzer\StatementsAnalyzer;
-use Psalm\Plugin\EventHandler\Event\FunctionReturnTypeProviderEvent;
-use Psalm\Plugin\EventHandler\FunctionReturnTypeProviderInterface;
-use Psalm\Type;
-use Psalm\Type\Atomic\TFalse;
-use Psalm\Type\Atomic\TLowercaseString;
-use Psalm\Type\Atomic\TNonEmptyString;
-use Psalm\Type\Atomic\TString;
-use Psalm\Type\Union;
-
-use function count;
-
-/**
- * @internal
- */
-class ExplodeReturnTypeProvider implements FunctionReturnTypeProviderInterface
```

Comment:
Haven't run tests for this: waiting for CI to tell me how bad this change will affect this patch :D